### PR TITLE
fix: webhook-name added multiple double quotes

### PR DESCRIPTION
**Pull request by hectorj2f**
Signed-off-by: hectorj2f <hectorf@vmware.com>

 **Description of the change**
 Double quotes were added twice  causing the following error:
 
 `error retrieving webhook: mutatingwebhookconfiguration.admissionregistration.k8s.io \"\\\"cosigned.sigstore.dev\\\"\" not found"`
 
 Fixes: https://github.com/sigstore/helm-charts/issues/165

 **Existing or Associated Issue(s)**

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-doc --dry-run` to preview the content
- [x] JSON Schema generated
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

**Review by vpnachev — DISMISSED** [review]
Thank for fixing it!

**Comment by cooktheryan** [comment]
:+1: thanks for putting this PR in it fixed my broken deploy!

File: charts/cosigned/templates/webhook/deployment_webhook.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ spec:
 {{- end }}
         args:
         {{- if semverCompare ">= 1.8-0" .Chart.AppVersion }}
-        - --webhook-name={{ required "A valid cosign.webhookName is required" .Values.cosign.webhookName | quote }}
+        - --webhook-name={{ required "A valid cosign.webhookName is required" .Values.cosign.webhookName }}
```
[the remark this answers is not in the document]
Comment (hectorj2f) [reply charts/cosigned/templates/webhook/deployment_webhook.yaml]: Done. Thanks

**Comment by vpnachev** [comment]
I guess the file `charts/cosigned/output` should be excluded from the PR?

**Review by vpnachev — APPROVED** [review]
/lgtm

**Review by cpanato — APPROVED** [review]

**Pull request merged**